packages/aws-fargate/package.json

```diff
@@ -68,9 +69,7 @@
     "@instana/metrics-util": "2.33.1",
     "@instana/serverless": "2.33.1"
   },
-  "devDependencies": {
-    "no-code2": "2.0.0"
-  },
+  "devDependencies": {},
```

Comment:
_nitpick:_ We could remove this section entirely, right? (Also in all the other packages where this is empty now.)

Reply:
Ok let's remove it 👍